provider/data/calendar/fingerprints.csv

```diff
@@ -1,11 +1,10 @@
-*, <'static strs>, 684B, 114 unique static strings
 calendar/chinesecache@1, <singleton>, 754B, 134f5ec7c9bdc494
 calendar/dangicache@1, <singleton>, 754B, 79d5d32449956411
 calendar/islamicobservationalcache@1, <singleton>, 504B, 18af87d272c76f14
 calendar/islamicummalquracache@1, <singleton>, 504B, ef6ac9ff14fd3f3
 calendar/japanese@1, <singleton>, 111B, b31e52deaf52706f
 calendar/japanext@1, <singleton>, 5216B, 6c20e216c8cd6e41
-datetime/week_data@1, <lookup>, 2760B, 115 identifiers
+datetime/week_data@1, <lookup>, 1626B, 115 identifiers
```

Comment:
Although these numbers are helpful, they are only proxy metrics. The real metric is the post-LTO binary size.